packages/geometry/src/vector3.ts

```diff
@@ -270,6 +272,30 @@ export function angleTo(a: Vector3, b: Vector3): number {
   return Math.acos(theta);
 }
 
+/**
+ * Returns the Euler angle, in radians with the `xyz` ordering, between two vectors.
+ */
+export function eulerTo(a: Vector3, b: Vector3): Euler.Euler {
+  const normalizedA = normalize(a);
+  const normalizedB = normalize(b);
+
+  const angle = Math.acos(dot(normalizedA, normalizedB));
+  const axis = normalize(cross(normalizedA, normalizedB));
+
+  const vectorsAreParallel = angle === Math.PI || angle === 0;
```

Comment:
Testing FP values for exact equality is an anti-pattern.

Reply:
I would write it  like this:

```
const dotDelta = Math.cos(some_very_small_angle); // e.g. 1 degree
const dotAB = dot(normalizedA, normalizedB);
const vectorsAreParallel = Math.abs(dotAB) > dotDelta;
```
... and don't try computing the cross-product until after you've confirmed the vectors aren't parallel.  The `crossAxisIsDefined` logic can be removed entirely.

In general, try to avoid the "arc cos" type functions as much as you can -- they aren't as precise as the other trig functions.